src/Components/Common/BedSelect.tsx

```diff
@@ -65,18 +69,18 @@ export const BedSelect = (props: BedSelectProps) => {
       selected={selected}
       onChange={setSelected}
       showNOptions={showNOptions}
-      placeholder="Search by beds name"
+      placeholder={t("Search by beds name")}
```

Comment:
Is this a valid translation key? See how translations are used in other places. Make sure the key is present in the translation files.